ios/features.mjs

```diff
@@ -60,16 +60,24 @@ export function isBridgelessEnabled(reactNativeVersion, options) {
  * @returns {boolean | "from-source"}
  */
 export function isHermesEnabled(platform, reactNativeVersion, options) {
-  const useHermes = process.env["USE_HERMES"];
-  const enabled =
-    typeof useHermes === "string"
-      ? useHermes === "1"
-      : options["hermesEnabled"] === true;
+  if (reactNativeVersion < v(0, 80, 0)) {
+    const useHermes = process.env["USE_HERMES"];
+    const enabled =
+      typeof useHermes === "string"
+        ? useHermes === "1"
+        : options["hermesEnabled"] === true;
 
-  // Hermes prebuilds for visionOS was introduced in 0.76
-  if (enabled && platform === "visionos" && reactNativeVersion < v(0, 76, 0)) {
-    return "from-source";
+    // Hermes prebuilds for visionOS was introduced in 0.76
+    if (
+      enabled &&
```

Comment:
It might be better to centralize these types of settings into capabilities. Maybe a function that returns an object with various capabilities given a RN version and process.env settings. It ends up being easier to manage. Doesn't need to be done now but something for you to consider.

Reply:
That's what this function is used for. See https://github.com/microsoft/react-native-test-app/blob/trunk/ios/app.mjs#L236-L255 and the tests in https://github.com/microsoft/react-native-test-app/blob/trunk/test/ios/app.test.ts